vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="container">
+    <img :src="backgroundImage" class="background" />
+    <div class="con-display">
+      <div class="v2-header-2-text text-mg">
+        We develop amazing products to help
+        <span class="canopas-gradient-text text-weight">Entrepreneurs </span>
+        bring their ideas to life.
+      </div>
+      <div class="img-center"><img :src="boyImage" /></div>
+    </div>
+    <div class="btn-mg">
+      <router-link to="">
+        <a class="v2-normal-3-text v2-black-btn btn-center"
+          >Let's talk
+          <font-awesome-icon class="arrow fa" icon="arrow-right" id="leftArrow"
+        /></a>
+      </router-link>
+    </div>
+  </div>
+</template>
+<script type="module">
+import backgroundImage from "@/assets/images/Landing/landingbackground.webp";
+import boyImage from "@/assets/images/Landing/Image.webp";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+
+export default {
+  data() {
+    return {
+      backgroundImage: backgroundImage,
+      boyImage: boyImage,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.container {
+  margin: 10rem;
```

Comment:
Remove this margin because landing section show in right side 

![Screenshot from 2022-04-20 12-04-26](https://user-images.githubusercontent.com/94360312/164166727-33d27430-f576-4c3a-a296-2b78f6c14850.png)